<a href="https://colab.research.google.com/github/LuisaMariaMontoyaConde1994/ESTADISTICAII/blob/main/Distribuciones_discretas.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

## **Nikol Sofia Perez Babativa.**
## **Paula Geraldine Morales Díaz.**

# **Distribuciones Discretas**

---



In [ ]:
import numpy as np
import matplotlib.pyplot as plt
from scipy.stats import binom
from scipy.stats import poisson
from scipy.stats import hypergeom
from scipy.stats import geom

In [ ]:
x=6
lamda=4
prob=poisson.pmf(x,lamda)
print(prob)

0.10419563456702102


In [ ]:
N=40
n=5
k=3
x=2
prob_1=hypergeom.pmf(x,N,n,k)
print(prob_1)

0.0354251012145749


# **Actividad**
---



# **Punto $1$**


Durante un experimento el promedio de particulas es de $4$ milisegundos
> ¿Cual es la probabilidad de que entre a lo más  $5$  particulas?

In [ ]:
lamda=4
t=1
x=5
prob_1=poisson.cdf(x,t,lamda)
print(prob_1)


0.7357588823428847


La probabilidad de que entre a lo más $5$ particulas  es de $73.57\%$ aproximadamente.

# **Punto $2$**

Lotes con $40$ componentes, cada uno que contenga $3$ o más defectuosos. Si se selecciona $5$ componentes al azar.
> ¿Cual es la probabilidad de que salga a lo más $2$ defectuosos?
$$(X\leq 2)$$


In [ ]:
N=40
n=5
k=3
x=2
prob_2=hypergeom.cdf(x,N,n,k)
print(prob_2)

0.9989878542510121


La probabilidad de que salga a lo más $2$ defectuosos es de $73.57\%$ aproximadamente.

# **Punto $3$**


Realice el mismo ejemplo de la binomial negativas usando poisson


En un campeonato, el equipo que gane $4$ de $7$ juegos será el ganador. Si se enfrentan los equipos $A$ y $B$, el equipo $A$ tiene una probabilidad de $55\%$ de ganarle al $B$.
> ¿Cual es la probabilidad de que gane el equipo $A$ en $6$ juegos?



In [ ]:
x=6-1
k=4-1
p=0.55
prob=binom.pmf(k,x,p)
print(prob)

0.3369093749999999


La probabilidad de que gane el equipo $A$ en $6$ juegos es de $33.69\%$ aproximadamente.

# **Punto $4$**

En un fabrica uno cada de $100$ articulos, en promedio resulta defectuoso.
> ¿Cual es la probabilidad de que $5$ articulos sea defectuoso?


In [ ]:
p=0.01
x=5
prob_4=geom.pmf(x,p)
print(prob_4)

0.0096059601


La probabilidad de que  $5$ articulos sea defectuoso es de $0.96\%$ aproximadamente.